bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractFormComponentImpl.java

```diff
@@ -445,4 +451,14 @@ public boolean isEmpty(SerializerProvider provider, String value) {
         }
     }
 
+    /**
+     * Fetches all the custom properties associated with a given component's instance (including additional custom properties)
+     *
+     * @return {@code Map<String, String>} returns all custom property key value pairs associated with the resource
+     */
+    private Map<String, String> getCustomProperties() {
+        return Optional.ofNullable(this.resource.adaptTo(CustomPropertyInfo.class))
```

Comment:
Once changes in far are merged and SDK is pushed, please update this PR so that the PR build passes

Reply:
ACK